### DISCUSSION TITLE
Does Nextflow use exit codes?

### DISCUSSION THREAD

**Discussion by stkgo** [nextflow-io/nextflow#5393] [Q&A] [answered]
Hi, I'm curious does Nextflow return different exit codes depending on how the pipeline execution concluded? For example, if the pipeline runs successfully will we see a 0 exit code and non-zero exit codes for failures?

**Comment by pditommaso** [comment, marked as answer]
Nextflow only uses `0` for success and `1` for (any) failure

**Reply by stkgo** [reply]
Perfect, thank you
